packages/client/src/utils/getOutcomeContent.tsx

```diff
@@ -108,9 +109,8 @@ const getOutcomeContent = (checker: imtr.Checker, slug: string) => {
       },
       [NEED_REPORT]: {
         mainContent: (
-          // @TODO: refactor these components, because we use the `NeedPermit` component to render the need report content, because it looks the same
           // Maybe rename this to `OutcomeLink` or `OutcomeMainContent`
-          <NeedPermit
+          <NeedReport
```

Comment:
NeedReport.test.tsx and NeedReport.tsx are now almost exact duplicates of NeedPermit.test.tsx and NeedPermit.tsx. It would be a lot nicer if you could refactor this to remove duplicate code. You can do this by merging them to one file, see comment here
```
// Maybe rename this to `OutcomeLink` or `OutcomeMainContent`
```

Reply:
@robinpiets Maybe we should only rename the component, outcomeContent?

Reply:
`OutcomeContent` is clearly already taken. The `NeedPermit` component (or atom?) is rendering an optional part of the total outcome with only a text and a button. Hence `OutcomeLink`